src/main/scala/io/smartdatalake/workflow/dataobject/HiveTableDataObject.scala

```diff
@@ -41,6 +41,8 @@ import scala.collection.JavaConverters._
  * @param table hive table to be written by this output
  * @param path hadoop directory for this table. If it doesn't contain scheme and authority, the connections pathPrefix is applied.
  *             If pathPrefix is not defined or doesn't define scheme and authority, default schema and authority is applied.
+ *             If DataObject is only used for reading or if the HiveTable already exist, the path can be omitted.
+ *             If the HiveTable already exists but with a different path, execution is stopped (unexpected behavior)
```

Comment:
this is only a warning now, isn't it?

Reply:
Right, changed it in https://github.com/smart-data-lake/smart-data-lake/pull/126/commits/c2cc545d39dd48874a514ddead4d15969613a4ca